java/src/jmri/jmrix/loconet/cmdstnconfig/CmdStnConfigPane.java

```diff
@@ -178,13 +173,22 @@ public void initComponents(LocoNetSystemConnectionMemo memo) {
                 gc.anchor = GridBagConstraints.WEST;
                 String label;
                 try {
-                    label = rb.getString("Option" + i);
+                    label = rb.getString("Option" + i); // model specific Option descriptions NOI18N
                     isReserved[i - MIN_OPTION] = false;
                 } catch (java.util.MissingResourceException e) {
-                    label = "" + i + ": " + rb.getString("Reserved");
+                    label = "" + i + ": " + Bundle.getMessage("Reserved");
                     isReserved[i - MIN_OPTION] = true;
                 }
                 JLabel l = new JLabel(label);
+                if (i > 20 && i < 24) {
+                    log.debug("CS name: {}", name);
+                    if (name.startsWith("DB150")) { // DB150 is the only model using different OpSw 21-23 combos than the common set in LocoNetBundle
+                        tooltip = rb.getString("DB150ConfigFxToolTip");
```

Comment:
Maybe put the tooltips in the CS-specific properties file, so they have be customized as needed?

Reply:
Similarly, as 90% looked identical, I chose to only override the DB150. But it's certainly a further refinement.

Reply:
For now, show an extra line in the tooltip to warn for specific versions:
![ln_cmdstnconfigopswtooltip_v2](https://user-images.githubusercontent.com/14811327/32700810-9e398d4c-c7ca-11e7-8dd9-c169be1393be.png)